appcues/src/main/java/com/appcues/ui/extensions/DrawScopeEx.kt

```diff
@@ -13,7 +13,7 @@ import androidx.compose.ui.unit.dp
 internal fun DrawScope.xShapePath(color: Color, deflateDp: Dp = 0.dp): Path {
     return Path()
         .apply {
-            val strokeWidth = 2.dp.toPx()
+            val strokeWidth = 1.5.dp.toPx()
```

Comment:
this will change the X on the screen capture confirmation dialog - but that is probably ok I think.

Reply:
it changes the x confirmation dialog, and I think its ok